pkg/virt-handler/vm_test.go

```diff
@@ -1914,7 +1914,7 @@ var _ = Describe("VirtualMachineInstance", func() {
 				UID:            "123",
 				StartTimestamp: &now,
 				EndTimestamp:   &now,
-				Completed:      true,
+				Failed:         false,
```

Comment:
This seems redundant

Reply:
@acardace Unresolved as I still see this in the commit.

Reply:
oops, let me fix it.

Reply:
Now it should be ok.